ProjetSupaero2018/catkin_ws/src/display/scripts/spawn_cylinder_server.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+import rospy
+from display.srv import SpawnCylinder, SpawnCylinderResponse
+import rospkg
+import subprocess
+from gazebo_msgs.srv import SpawnModel
+from geometry_msgs.msg import Pose, Point, Quaternion
+
+
+# rosrun xacro xacro.py model.xacro > model.urdf
+PACKAGE_NAME = 'display'
+ROS_PACKAGE = rospkg.RosPack()
+PACKAGE_DIR = ROS_PACKAGE.get_path(PACKAGE_NAME)
+XACRO_PATH = PACKAGE_DIR + '/xacro/'
+URDF_PATH = PACKAGE_DIR + '/urdf/'
+model_count = 0
+
+
+def spawn_cylinder(req):
```

Comment:
This function might definitely be cut in half (create urdf + spawn cylinder) for the sake of clarity